sdk/src/main/java/com/microsoft/hydralab/performance/PerformanceInspector.java

```diff
@@ -1,10 +1,13 @@
 package com.microsoft.hydralab.performance;
 
 public interface PerformanceInspector {
-    String INSPECTOR_ANDROID_MEMORY_DUMP = "AndroidMemoryDump";
-    String INSPECTOR_ANDROID_MEMORY_INFO = "AndroidMemoryInfo";
-    String INSPECTOR_ANDROID_BATTERY_INFO = "AndroidBatteryInfo";
-    String INSPECTOR_WIN_BATTERY = "WindowsBattery";
-    String INSPECTOR_WIN_MEMORY = "WindowsMemory";
+    enum PerformanceInspectorName {
```

Comment:
PerformanceInspector**Type** is better for me than PerformanceInspector**Name**. It is up to you, if you would like to choose the current name, just resolve this conversation.

Reply:
Updated